packages/polymath-shared/src/constants.js

```diff
@@ -58,22 +66,26 @@ const localAddresses = {
   PercentageTransferManagerFactory:
     PercentageTransferManagerFactoryArtifacts.networks[LOCAL_NETWORK_ID]
       .address,
+  CountTransferManagerFactory:
+    CountTransferManagerFactoryArtifacts.networks[LOCAL_NETWORK_ID],
 };
 
 const kovanStagingPolyFaucetAddress =
-  '0x33c0c74675bb3e8fbf7d7510da998c3ef913e407';
+  '0xe2b733524f1a630519d1b14811cafffd41ad26e5';
 const kovanStagingAddresses = {
   PolyTokenFaucet: kovanStagingPolyFaucetAddress,
   PolyToken: kovanStagingPolyFaucetAddress,
-  PolymathRegistry: '0x9a987a1eee04c2784030347df6778e1a9170f467',
-  TickerRegistry: '0x6665a9bc33edc5839907f06c4c4586a4117aafb2',
-  ModuleRegistry: '0xf661fa176636c7b97d281bc802b0da8d43851ec4',
-  SecurityTokenRegistry: '0x7edd08ddc4763d6243cab427721d2afcca843579',
-  CappedSTOFactory: '0xdeacac8248723dbaf2bf6776f293228993e05a0e',
-  GeneralPermissionManagerFactory: '0xf97976bab461f354a06aa30132f56278c33c86bb',
-  CountTransferManagerFactory: '0xb36e11e8b01811610a04fccec25651566438109b',
+  PolymathRegistry: '0x40b53ebd7e0d4a5cf7112371e2552eb153e1087c',
+  TickerRegistry: '0x762af9b4d8c9e4affb92091479ce0524cd66001c',
+  ModuleRegistry: '0x3ac848b3e94b78d5acc3f8e3c41cae92d59e2bfd',
+  SecurityTokenRegistry: '0xeae834e007af3c081722f7f97003fca321c1470e',
+  CappedSTOFactory: '0xf2cc7915fe48d9a709a1199cb018dcc27c114946',
+  GeneralPermissionManagerFactory: '0x08b056dfd4f0b5e9ed8f6dfbff3163107384b857',
+  CountTransferManagerFactory: '0x93b6ef91e4ea396594c18c88de5d31c566cabdf3',
   PercentageTransferManagerFactory:
-    '0x25b7e309441e47c1892b56fa420bda02afdaa14f',
+    '0xeb3e23b507ed8d71ef9b0a7ca31af6a4e20c0143',
+  USDTieredSTOFactory:
```

Comment:
Why is there an address for this contract if USDTieredSTO isn't supported in 1.3.3?

Reply:
Fixed, leaked this into the commit by accident